pkgs/applications/networking/irc/kvirc/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub
+, qt5
+, pkgconfig, cmake, gettext
+}:
+
+stdenv.mkDerivation rec {
+  name = "kvirc-${version}";
```

Comment:
Using `pname = "kvirc";` will automatically set `name = "${pname}-${version}";` for you, so please replace `name` with `pname`.